Makefile

```diff
@@ -3,62 +3,173 @@
 ##################################################
 IMAGE_TAG      ?= master
 IMAGE_REGISTRY ?= docker.io
-IMAGE_REPO     ?= kedacore
+# IMAGE_REPO     ?= kedacore
+IMAGE_REPO     ?= samuelmacko
 
 IMAGE_CONTROLLER = $(IMAGE_REGISTRY)/$(IMAGE_REPO)/keda-olm-operator:$(IMAGE_TAG)
 
 
 ARCH       ?=amd64
 CGO        ?=0
 TARGET_OS  ?=linux
+VERSION ?= 0.0.1
 
 GIT_VERSION = $(shell git describe --always --abbrev=7)
 GIT_COMMIT  = $(shell git rev-list -1 HEAD)
 DATE        = $(shell date -u +"%Y.%m.%d.%H.%M.%S")
 
+ifneq ($(origin CHANNELS), undefined)
+BUNDLE_CHANNELS := --channels=$(CHANNELS)
+endif
+ifneq ($(origin DEFAULT_CHANNEL), undefined)
+BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
+endif
+
+GO_BUILD_VARS= GO111MODULE=on CGO_ENABLED=$(CGO) GOOS=$(TARGET_OS) GOARCH=$(ARCH)
+
+# Current Operator version
+# Default bundle image tag
+BUNDLE_IMG ?= controller-bundle:$(VERSION)
+# Options for 'bundle-build'
+BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
+
+# Image URL to use all building/pushing image targets
+IMG ?= controller:latest
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+CRD_OPTIONS ?= "crd:trivialVersions=true"
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
+
 ##################################################
 # All                                            #
 ##################################################
-.PHONY: All
+# all: manager
 all: build
 
-
 ##################################################
 # PUBLISH                                        #
 ##################################################
-.PHONY: publish
-publish: build
+publish: docker-build
+	# docker push $(IMAGE_ADAPTER)
 	docker push $(IMAGE_CONTROLLER)
 
-.PHONY: dev
-dev:
-	$(GO_BUILD_VARS) operator-sdk build $(IMAGE_CONTROLLER) \
-		--go-build-args "-ldflags -X=main.GitCommit=$(GIT_COMMIT) -o build/_output/bin/keda-olm-operator"
-	docker push $(IMAGE_CONTROLLER)
+##################################################
+# Release                                        #
+##################################################
+.PHONY: release
+release: manifests kustomize
```

Comment:
That's not relevant here. It is for KEDA not KEDA OLM Operator.

I recommend you to generate the Makefile once again and the only modify the necessary parts